### DISCUSSION TITLE
Download

### DISCUSSION THREAD

**Discussion by FdMach** [collin80/SavvyCAN#858] [Q&A]
Why is it so difficult to download Savvycan  ?
I have visited the site a half a dozen times and can't find where to download

**Comment by collin80** [comment]
It may unfortunately be more confusing than it should be.

The best place to download binaries is from github itself -> https://github.com/collin80/SavvyCAN/releases

There you can get the latest bleeding edge builds for both QT5 and QT6, as well as the latest "Official" build which is quite old these days. In general, the latest build of the master branch may be the best bet.
